Image resizer hot reload [#237]

AdityaSriram09 opened the pull request:
Fix for https://github.com/localstack/localstack-docs/issues/237
@blkgrlcto please review it

AdityaSriram09 committed:
Create Serverless Image Resizer with Hot Reload.md #237

https://github.com/localstack/localstack-docs/issues/237

AdityaSriram09 committed:
Rename

AdityaSriram09 committed:
Delete src/content/docs/aws/tutorials/S3-Image-Resizer-with-Hot-Reload.md

AdityaSriram09 committed:
Create image-resizer-hot-reload.md

https://github.com/localstack/localstack-docs/issues/237

AdityaSriram09 committed:
Update schema-evolution-glue-msk.mdx [#229]

Add - Testing the application

AdityaSriram09 committed:
Merge branch 'localstack:main' into main

AdityaSriram09 committed:
Remove testing section for MSK and Glue integrations

Removed the testing section for the application

blkgrlcto reviewed (changes requested):
Thanks for adding this! The content is helpful, but the formatting feels a bit uneven and the section reads a bit bare. I’d suggest tightening up the markdown structure and expanding with a short “why/use case” blurb or example to give it more context before merging.

AdityaSriram09 committed:
Merge branch 'localstack:main' into image-resizer-hot-reload

AdityaSriram09 committed:
Merge branch 'localstack:main' into image-resizer-hot-reload

AdityaSriram09 committed:
Enhance image resizer tutorial with use cases and clarity

Added a new section on use cases for image resizing and updated the steps for clarity. Enhanced the document structure with additional headers and improved descriptions.

remotesynth committed:
Fix issues with frontmatter
